pages/home.js

```diff
@@ -1,19 +1,78 @@
 import PropTypes from 'prop-types'
 import en from '../locales/en'
 import fr from '../locales/fr'
+import HomeCardContainer from '../components/HomeCardContainer'
+import TextInput from '../components/TextInput'
 
 import { fetchContent } from '../lib/cms'
 
 export default function Home(props) {
   /* istanbul ignore next */
   const t = props.locale === 'en' ? en : fr
+
+  const handleCreateSubmit = (e) => {
+    e.preventDefault()
+    console.log(e.target.owner.value)
+
+    //TODO: Make call to back end to get random room id.
+    //TODO: Redirect user to that room id.
+    //  router.push({
+    //    pathname: "/room/1",
+    //    query: {q: "test"},
+    //  })
+  }
+
+  const handleJoinSubmit = (e) => {
+    e.preventDefault()
+  }
+
   return (
     <div
+      data-testid="homeContent"
       id="homeContent"
-      className="container mx-auto px-6 mt-5 bg-slate-300 p-8"
+      className="container grid grid-cols-1 gap-y-5 mx-auto sm:flex sm:justify-around"
```

Comment:
```suggestion
      className="container grid grid-cols-1 gap-5 mx-auto sm:flex sm:justify-center"
```

To bring the containers into the center rather than having the huge gap between them, to look more like the Figma designs